web/libs/editor/src/components/BottomBar/buttons.tsx

```diff
@@ -15,15 +15,15 @@ type MixedInParams = {
   history: any;
 };
 
-export function controlsInjector<T extends {}>(fn: (props: T & MixedInParams) => ReactNode) {
+export function controlsInjector<T extends {}>(fn: (props: T & MixedInParams) => ReactElement) {
   const wrapped = inject(({ store }) => {
     return {
       store,
       history: store?.annotationStore?.selected?.history,
     };
   })(fn);
   // inject type doesn't handle the injected props, so we have to force cast it
-  return wrapped as unknown as (props: T) => ReactNode;
+  return wrapped as unknown as (props: T) => ReactElement;
```

Comment:
you don't need to specify return type in the function signature if you explicitly do that in the return statement

Reply:
At first a thought that you are right but then I figured out that it's different variables.
<img width="873" alt="image" src="https://github.com/user-attachments/assets/85fce9e9-048d-460e-b23a-8c0ae899a951">